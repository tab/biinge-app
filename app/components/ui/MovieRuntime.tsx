import React from "react"

import { formatRuntime } from "helpers/formatRuntime"
import Typography from "components/ui/Typography"
import { movieStyles } from "styles"

type Props = {
  children: number
}

const MovieReleaseDateComponent = ({ children }: Props) => {
  return (
    <Typography variant="subhead" style={movieStyles.runtime}>
      {formatRuntime(children)}
    </Typography>
  )
}

export default MovieReleaseDateComponent
