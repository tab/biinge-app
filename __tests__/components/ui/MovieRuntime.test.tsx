import React from "react"
import "react-native"
import { render } from "@testing-library/react-native"
import { it } from "@jest/globals"

import MovieRuntime from "components/ui/MovieRuntime"

describe("MovieRuntime", () => {
  const runtime = 124

  it("renders correctly", () => {
    const tree = render(<MovieRuntime>{runtime}</MovieRuntime>)
    expect(tree).toMatchSnapshot()
  })
})
