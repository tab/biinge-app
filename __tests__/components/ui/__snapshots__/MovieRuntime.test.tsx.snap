// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieRuntime renders correctly 1`] = `
<Text
  ellipsizeMode="tail"
  numberOfLines={0}
  style={
    [
      {
        "color": "#181818",
        "fontSize": 16,
        "fontWeight": "700",
        "lineHeight": 20,
      },
      {
        "color": "#999999",
      },
    ]
  }
>
  2h4m
</Text>
`;
